docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -162,3 +162,19 @@ test('Close a specific window', async t => {
     await t.closeWindow(window1);
 });
 ```
+
+## Limitations
+
+This section describes the limitations of multiple browser windows testing in TestCafe.
+
+### Limited Video Recording
```

Comment:
Maybe we can come up with a less broad heading, like `Video Aspect Ratio Notes/Limitations/...`?